node-hub/feetech-client/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "feetech-client"
+version = "0.1"
+authors = ["Hennzau <dev@enzo-le-van.fr>"]
+description = "Dora Node client for feetech motors."
+readme = "README.md"
+
+packages = [
+    { include = "main.py", to = "feetech_client" }
+]
+
+[tool.poetry.dependencies]
+dora-rs = "0.3.5"
+
+common = { path = "../../common" }
```

Comment:
Same remark. I think it could be great to later have those feetech client be standalone pip package that people can reuse in their own project.